src/ch09-02-recoverable-errors-with-result.md

```diff
@@ -136,7 +136,7 @@ copy and paste relevant text
 -->
 
 ```text
-thread 'main' panicked at 'hello.txt should be included in this project: Os {
+thread 'main' panicked at 'hello.txt 應該要存在此專案中: Os {
```

Comment:
Why changed this?

IIRC l10n in rust compiler diagnostics is not yet rolled out

Reply:
ok. I originally thought the message should be matched with the [code](https://github.com/rust-tw/book-tw/blob/zh_tw/listings/ch09-error-handling/no-listing-05-expect/src/main.rs).

Reply:
My fault. You're absolutely right. I thought it was a message from rustc.